extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -140,33 +155,30 @@ public function validate( $regex ) {
 	 *
 	 */
 	public function stats() {
-		wfProfileIn( __METHOD__  );
-		$result = $this->sendToPhalanxDaemon( "stats", array() );
-		wfProfileOut( __METHOD__  );
+		wfProfileIn( __METHOD__ );
+		$result = $this->sendToPhalanxDaemon( "stats", [] );
+		wfProfileOut( __METHOD__ );
 		return $result;
 	}
 
 	/**
 	 * Send prepared request request to phalanx daemon
 	 *
-	 * @author Krzysztof Krzyżaniak (eloy) <eloy@wikia-inc.com>
-	 * @access private
-	 *
-	 * @param $action String type of action
-	 * @param $parameters Array additional parameters as hash table
-	 * @return integer|mixed data of blocks applied or numeric value (0 - block applied, 1 - no block applied)
+	 * @param string $action String type of action
+	 * @param array $parameters additional parameters as hash table
+	 * @return integer|mixed data of blocks applied or numeric value
+	 *         (0 - block applied, 1 - no block applied)
 	 */
 	private function sendToPhalanxDaemon( $action, $parameters ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "sendToPhalanxDaemon" is 27 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "sendToPhalanxDaemon" has 159 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)